pykube/config.py

```diff
@@ -152,7 +157,11 @@ def user(self):
         """
         if self.current_context is None:
             raise exceptions.PyKubeError("current context not set; call set_current_context")
-        return self.users[self.contexts[self.current_context]["user"]]
```

Comment:
Minor style change--Was having some trouble quickly reading/understanding the line so I broke it out into some temp explaining variables.  Been a while, but might have also been due to an invalid key somewhere, so breaking it out made it easier to figure out where it was blowing up when running.  Doesn't sound like it is an improvement, so I can change it back. :)